### DISCUSSION TITLE
World map converted without house numbers

### DISCUSSION THREAD

**Discussion by ioantiba** [onthegomap/planetiler#579] [Q&A] [answered]
Hello everyone. 
I managed to convert the OSM world map into .mbtiles but I don't have the number of buildings (housenumber).
I used the following command to convert the map:

java -Xmx120g \
   -jar planetiler.jar \
   -XX:MaxHeapFreeRatio=40 \
   --area=planet --bounds=world \
   --osm-path=/home/maps/planet-latest.osm.pbf
   --fetch-wikidata \
   --output=planet.mbtiles \
   --nodemap-type=sparsearray --nodemap-storage=ram

I previously rendered the map of Europe using tilemaker and there I can see the house numbers, but the time and resources used by that tool are much greater.

Can someone help me with information about how I could convert the map into .mbtiles and make the house numbers available?

Thanks in advance for any help.

**Comment by acalcutt** [comment, marked as answer]
housenumber should be there with those options. For example, if I look at my planetiler generated file in tileserver-gl inspect, I can see there is a housenumber layer and I can see the points in each building that has one.
![image](https://github.com/onthegomap/planetiler/assets/3792408/ae559ff9-2939-4fba-9248-bfea797490b4)

What does your style look like? I use something like this
https://github.com/acalcutt/wifidb-tileserver-gl/blob/master/tileserver-gl/styles/WDB_OSM/style.json#L4438-L4459
which shows them after zoom level 17
![image](https://github.com/onthegomap/planetiler/assets/3792408/53392258-293e-49f5-8939-e225364e1741)

**Reply by ioantiba** [reply]
Hello,
Indeed, you are right, the house numbers are there.
I use the style "https://github.com/openmaptiles/osm-bright-gl-style" and after looking in style.json it seems that there is no section for "housenumber".
I was misled by the fact that before I had the European map rendered with tilemaker and I used the same style.json and the house numbers were displayed there. However, after rendering with planetiler, they were no longer visible.
Now I changed the style used and everything is fine.

Thank you very much for your help!
